Remove the dependencies so we do not try fetching them
again in do_build. It seems ant has no way to do this.

--- a/build.xml
+++ b/build.xml
@@ -172,8 +172,7 @@
 			  value="true"/>
 	</target>
 
-	<target name="setup"
-		depends="init,retrieve">
+	<target name="setup">
 		<taskdef resource="net/sf/antcontrib/antlib.xml"
 			 uri="antlib:net.sf.antcontrib">
 			<classpath>
@@ -288,7 +287,7 @@
 	</target>
 
 	<target name="compile-test"
-		depends="init,retrieve,compile">
+		depends="compile">
 		<mkdir dir="${classes.dir}/test"/>
 		<depend srcDir="${basedir}/test"
 			destDir="${classes.dir}/test"
@@ -473,7 +472,7 @@
 	</target>
 
 	<target name="test"
-		depends="init,retrieve,compile,compile-test"
+		depends="compile,compile-test"
 		description="run unit tests">
 		<delete dir="${build.dir}/test/raw-reports"/>
 		<mkdir dir="${build.dir}/test/raw-reports"/>
@@ -504,7 +503,7 @@
 	</target>
 
 	<target name="build"
-		depends="init,retrieve,setup,compile"
+		depends="setup,compile"
 		description="build the jEdit JAR-file with full debug-information">
 		<mkdir dir="${jar.location}"/>
 		<jar destfile="${jar.location}/${jar.filename}"
@@ -562,7 +561,7 @@
 	</target>
 
 	<target name="build-exe-launcher"
-		depends="init,retrieve,setup,filter-package-files"
+		depends="setup,filter-package-files"
 		description="build the EXE launcher">
 		<contrib:if>
 			<os family="windows"/>
@@ -692,8 +691,7 @@
 		</java>
 	</target>
 
-	<target name="unpack-docbook-xsl"
-		depends="init,retrieve">
+	<target name="unpack-docbook-xsl">
 		<unzip src="${lib.dir}/docbook/docbook-xsl-resources.zip"
 		       dest="${build.dir}/docbook-xsl/"/>
 	</target>
@@ -739,7 +737,7 @@
 	</presetdef>
 
 	<target name="generate-doc-faq"
-		depends="init,retrieve,setup,unpack-docbook-xsl">
+		depends="setup,unpack-docbook-xsl">
 		<dependset.html>
 			<srcfileset dir="doc/FAQ"/>
 			<targetfileset dir="${jar.location}/doc/FAQ"/>
@@ -763,7 +761,7 @@
 	</target>
 
 	<target name="generate-doc-news"
-		depends="init,retrieve,setup,unpack-docbook-xsl">
+		depends="setup,unpack-docbook-xsl">
 		<dependset.html>
 			<srcfileset dir="doc/whatsnew"/>
 			<targetfileset dir="${jar.location}/doc/whatsnew"/>
@@ -787,7 +785,7 @@
 	</target>
 
 	<target name="generate-doc-users-guide"
-		depends="init,retrieve,setup,unpack-docbook-xsl">
+		depends="setup,unpack-docbook-xsl">
 		<dependset.html>
 			<srcfileset dir="doc/users-guide"/>
 			<targetfileset dir="${jar.location}/doc/users-guide"/>
@@ -912,7 +910,7 @@
 	</target>
 
 	<target name="generate-pdf-users-guide"
-		depends="init,retrieve,setup,unpack-docbook-xsl">
+		depends="setup,unpack-docbook-xsl">
 		<fail message="Please set the property &quot;paper.type&quot;"
 		      unless="paper.type"/>
 		<contrib:switch value="${paper.type}">
@@ -1216,7 +1214,7 @@
 	</target>
 
 	<target name="dist-mac-finish"
-		depends="init,retrieve,setup"
+		depends="setup"
 		description="finish building the Mac OS X disk image (DMG-file) on Mac OS X">
 		<fail message="The disk image (DMG-file) for easy distribution on Mac OS X can only be built on Mac OS X currently">
 			<condition>
@@ -1342,7 +1340,7 @@
 	</target>
 
 	<target name="dist-mac"
-		depends="init,retrieve,setup,prepare-dist-files"
+		depends="setup,prepare-dist-files"
 		description="build the Mac OS X disk image (DMG-file)">
 		<contrib:if>
 			<os family="mac"/>
@@ -1449,7 +1447,7 @@
 	</target>
 
 	<target name="prepare-dist-files"
-		depends="init,retrieve,setup">
+		depends="setup">
 		<antcall target="build">
 			<param name="config.build.debuglevel"
 			       value="lines,source"/>
@@ -1627,7 +1625,7 @@
 	</target>
 
 	<target name="dist-deb"
-		depends="init,retrieve,setup,prepare-dist-files"
+		depends="setup,prepare-dist-files"
 		description="build the DEB Package">
 		<antcall target="compile-ar">
 			<param name="config.build.debuglevel"
@@ -1873,7 +1871,7 @@
 	</target>
 
 	<target name="dist-sign-deb-Release"
-		depends="init,retrieve,setup"
+		depends="setup"
 		description="sign the DEB Release file">
 		<contrib:if>
 			<not>
